### DISCUSSION TITLE
GUI 1.2.2 does not automatically detect plots created with CLI (main machine + harvesters)

### DISCUSSION THREAD

**Discussion by JimbiniBambini** [Chia-Network/chia-blockchain#7697] [Support]
Hey guys, 

I would be so happy if you would add a feature (or fix a BUG), which automatically detects new plots added by CLI to the specified directories (same directory in GUI and CLI).

Curently everything what was created with CLI on the same PC, where the main node is running and on the harvester must be updated manually over the GUI (refresh plots --> after they are added). Even HPOOL mining SW had this feature, as the specified folders were scanned cyclically and there is no need to refresh the plots manually.

Thank a lot!

**Comment by keliew** [comment]
You mean `plot_loading_frequency_seconds` in `config.yaml`?

**Reply by JimbiniBambini** [reply]
wooooaaaa, let me check it :)

**Reply by keliew** [reply]
> wooooaaaa, let me check it :)

2 mins of waiting time for a plot refresh, I think is OK...

Unless you mean you found a bug..?

**Reply by JimbiniBambini** [reply]
Checked it --> it's set to 120s as you mentioned, however no plots are being added. Take in account, that I use completely the CLI for plotting, so the GUI seems to have an issue with this setting unless you start the plots directly from it.

**Comment by FVierra27** [comment]
Same problem here. I plot on remote plotters and rsync to farming plot directory. GUI doesn't update plot count unless I hit refresh plots manually. It was working on previous versions.

**Comment by JimbiniBambini** [comment]
So, dear community, any updates/news from Devs related to this problem? It's not possible, that only me and one more user experience the same problem. Otherwise simply give us a hint, what we may be doing wrong. The suggested parameter in config file is set to 120s, but no new plots are added to the list, even though I create them in a "while 1" loop over CLI.

**Comment by emlowe** [comment]
It is a known issue see #7262 

We are working on many fixes in this area. This will help PR #7744. But we expect more PRs as well.
